preprocessing/nextclade/src/loculus_preprocessing/prepro.py

```diff
@@ -390,21 +398,26 @@ def add_input_metadata(
     nextclade_prefix = "nextclade."
     if input_path.startswith(nextclade_prefix):
         segment = spec.args.get("segment", "main")
-        if not unprocessed.nextcladeMetadata:
```

Comment:
this case will now never occur as nextcladeMetadata always includes a dictionary, potentially though a dictionary with empty values

Reply:
An entirely empty dictionary will be trigger this branch afaik

Reply:
That is true but we should never have an empty dictionary and get to this function, as if there existed an unaligned sequence for that segment the dictionary will now have segmentName as a key. Otherwise if there were no unaligned sequences even process_single will break early and we will get to this function. However, I can keep this case and just mark in the warning that this case should never happen and that users should contact the administrator.